src/App.tsx

```diff
@@ -1,7 +1,102 @@
-import "./App.css";
+import { ClipboardEvent, KeyboardEvent, useState } from "react";
+import { Box, Button, TextField, Typography } from "@mui/material";
+import { getLastDigit, saveData } from "./axios/useAxios";
+import { handleClear, handleInput, handlePaste } from "./utils/functions";
 
 function App() {
-  return <></>;
+  const NUM_OF_DIGITS = 8;
+
+  const [values, setValues] = useState<string[]>(
+    new Array(NUM_OF_DIGITS).fill("")
+  );
+  const [lastDigit, setLastDigit] = useState<string | undefined>(undefined);
+
+  const handleClick = async () => {
+    const lastDigit: string | undefined = await getLastDigit(values.join(""));
+
+    if (lastDigit) {
+      setLastDigit(lastDigit);
+
+      await saveData(values.join(""), lastDigit.toString());
+    }
+  };
+
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        mt: "25vh",
+        gap: "3vh",
+      }}
```

Comment:
please add Style.ts file and import styles from there and not hard coded in the component